pkg/protocol/http/http.go

```diff
@@ -22,7 +23,6 @@ import (
 
 	cloudevents "github.com/cloudevents/sdk-go/v2"
 	httpClient "github.com/cloudevents/sdk-go/v2/client"
-	cehttp "github.com/cloudevents/sdk-go/v2/protocol/http"
 	httpP "github.com/cloudevents/sdk-go/v2/protocol/http"
```

Comment:
this is cloudevents http protocol used , will it help if we keep it  as cehttp

Reply:
Can I remove the httpP then? Otherwise it complains about duplicate import

Reply:
yes lets have it as cnehttp